constructor/main.py

```diff
@@ -283,7 +283,7 @@ def main():
                    action="store",
                    default=cc_platform,
                    help="the platform for which installer is for, "
-                   "defaults to '{}'".format(cc_platform))
+                   f"defaults to '{cc_platform}'. Options: {SUPPORTED_PLATFORMS}")
```

Comment:
Can we add a `e.g.` to state that this is not limited to the given options here?:
```suggestion
                   f"defaults to '{cc_platform}'. Options, e.g.: {SUPPORTED_PLATFORMS}")
```